app/controllers/review_controller.rb

```diff
@@ -5,7 +5,19 @@ class ReviewController < ApplicationController
   before_action :require_unreleased_feature_permissions!
   before_action :set_step_by_step_page
 
-  def submit_for_2i; end
+  def submit_for_2i
+    if request.post?
+      if @step_by_step_page.update(
+        review_requester: current_user.name,
```

Comment:
Instead of storing the actual name of the requester, would it be cleaner to store the user's ID, then retrieve their name from the `users` table when displayed?

It would also future-proof this, e.g. if we ever wanted to have the email address of the requester displayed.

Reply:
It would. I was following the pattern that's used for setting "assigned_to", though that's no reason not to make it better here.

Reply:
Looks like the same pattern has been used previously in the change notes too.  It's not something I'm massively keen on since it feels like we're duplicating data (and then showing out of date names when someone's name has changed).